source/extensions/clusters/redis/redis_cluster.cc

```diff
@@ -297,7 +297,7 @@ void RedisCluster::RedisDiscoverySession::startResolveRedis() {
     client->host_ = current_host_address_;
     client->client_ = client_factory_.create(host, dispatcher_, *this, redis_command_stats_,
                                              parent_.info()->statsScope(), parent_.auth_username_,
-                                             parent_.auth_password_, false,false,nullptr);
+                                             parent_.auth_password_, false,false,false,nullptr);
```

Comment:
we can try to optimise this by accepting an enum as input, and setting booleans appropriately

Reply:
Ok ji